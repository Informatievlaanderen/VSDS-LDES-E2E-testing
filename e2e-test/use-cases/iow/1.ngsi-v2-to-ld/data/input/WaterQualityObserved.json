{
    "id": "urn:ngsi-v2:cot-imec-be:WaterQualityObserved:dwg-iow-886nJGdroD857YjumSEuNj",
    "type": "WaterQualityObserved",
    "conductivity": {
        "type": "Number",
        "value": 6150,
        "metadata": {
            "unitCode": {
                "type": "text",
                "value": "G42"
            }
        }
    },
    "dateObserved": {
        "type": "DateTime",
        "value": "2022-09-09T09:10:00.000Z",
        "metadata": {}
    },
    "depth": {
        "type": "Number",
        "value": 1442,
        "metadata": {
            "unitCode": {
                "type": "text",
                "value": "HP"
            }
        }
    },
    "location": {
        "type": "geo:json",
        "value": {
            "type": "Point",
            "coordinates": [
                2.854459744,
                51.03321207
            ]
        },
        "metadata": {
            "name": {
                "type": "Text",
                "value": "loc-00019-33"
            }
        }
    },
    "locationName": {
        "type": "Text",
        "value": "loc-00019-33",
        "metadata": {}
    },
    "refDevice": {
        "type": "Relationship",
        "value": "urn:ngsi-v2:cot-imec-be:Device:dwg-iow-JAmXrtiSYwBpsbN9s4nQDG",
        "metadata": {}
    },
    "refDevices": {
        "type": "StructuredValue",
        "value": [
            "urn:ngsi-v2:cot-imec-be:Device:dwg-iow-JAmXrtiSYwBpsbN9s4nQDG"
        ],
        "metadata": {}
    },
    "samplingPoint": {
        "type": "Text",
        "value": "spt-00035-79",
        "metadata": {}
    },
    "temperature": {
        "type": "Number",
        "value": 20.43,
        "metadata": {
            "unitCode": {
                "type": "text",
                "value": "CEL"
            }
        }
    }
}
