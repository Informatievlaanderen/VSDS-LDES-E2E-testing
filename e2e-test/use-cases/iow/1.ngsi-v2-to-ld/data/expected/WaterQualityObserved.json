{
    "@context": [
        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"
    ],
    "id": "urn:ngsi-v2:cot-imec-be:WaterQualityObserved:dwg-iow-886nJGdroD857YjumSEuNj",
    "type": "WaterQualityObserved",
    "locationName": {
        "type": "Property",
        "value": "loc-00019-33",
        "observedAt": "2022-09-09T09:10:00.000Z"
    },
    "refDevices": {
        "type": "Property",
        "value": "urn:ngsi-v2:cot-imec-be:Device:dwg-iow-JAmXrtiSYwBpsbN9s4nQDG",
        "observedAt": "2022-09-09T09:10:00.000Z"
    },
    "samplingPoint": {
        "type": "Property",
        "value": "spt-00035-79",
        "observedAt": "2022-09-09T09:10:00.000Z"
    },
    "location": {
        "type": "GeoProperty",
        "value": {
            "type": "Point",
            "coordinates": [
                2.854459744,
                51.03321207
            ]
        },
        "name": {
            "type": "Property",
            "value": "loc-00019-33"
        },
        "observedAt": "2022-09-09T09:10:00.000Z"
    },
    "conductivity": {
        "type": "Property",
        "value": 6150,
        "observedAt": "2022-09-09T09:10:00.000Z",
        "unitCode": "G42"
    },
    "https://uri.fiware.org/ns/data-models#dateObserved": {
        "type": "Property",
        "value": {
            "type": "DateTime",
            "@value": "2022-09-09T09:10:00.000Z"
        },
        "observedAt": "2022-09-09T09:10:00.000Z"
    },
    "depth": {
        "type": "Property",
        "value": 1442,
        "observedAt": "2022-09-09T09:10:00.000Z",
        "unitCode": "HP"
    },
    "refDevice": {
        "type": "Relationship",
        "object": "urn:ngsi-v2:cot-imec-be:Device:dwg-iow-JAmXrtiSYwBpsbN9s4nQDG"
    },
    "temperature": {
        "type": "Property",
        "value": 20.43,
        "observedAt": "2022-09-09T09:10:00.000Z",
        "unitCode": "CEL"
    }
}